src/libs/ajax/leonardo/models/app-models.ts

```diff
@@ -75,10 +75,11 @@ export interface ListAppResponse {
   errors: AppError[];
   status: LeoAppStatus;
   proxyUrls: Record<string, string>;
-  diskName?: string;
+  diskName?: string | null;
```

Comment:
In dev, this field is null for a WDS app.

Reply:
the backend model is defined as Option[DiskName]... it seems the decoder implementation of putting null in an Optional field may not be compatible with typescript's expectation about optional fields 🤔